modules/H2store/examples/ExamplePVTGenerationH2Brine.m

```diff
@@ -0,0 +1,83 @@
+% MRST Example: Setting Up H₂-Brine Fluid Properties for Black-Oil Simulation
+%
+%{
+Copyright 2009-2025 SINTEF Digital, Mathematics & Cybernetics.
+
+This file is part of The MATLAB Reservoir Simulation Toolbox (MRST).
+
+MRST is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+MRST is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with MRST.  If not, see <http://www.gnu.org/licenses/>.
+%}
+% Parameters for Temperature, Pressure, and Saturation Range
+min_temp = 40;                % Minimum temperature in Celsius
+max_temp = 60;                % Maximum temperature in Celsius
+min_pressure = 1 * atm;       % Minimum pressure in Pa
+max_pressure = 45 * barsa();      % Maximum pressure in Pa
+nbp = 10;                     % Number of pressure points
+nbt = 10;                     % Number of temperature points
+ms = 0;                       % Salt molality [mol/kg]
+%% Notice on Computational Cost
+warning('ComputationalCost:Medium', ...
+       ['Please be advised that for large nbp and nbt this example often takes a long time ', ...
+        'to run: this script will extract datan from https://webbook.nist.gov']);
+
+currentDir = fileparts(mfilename('fullpath'));
+outputDisplay = false;
+% Define the target output directory relative to the current directory
+output_dir = fullfile(currentDir, 'PVT', 'PVT_H2STORAGE');
```

Comment:
You should use the mrstOutputDirectory() variable here instead:
```suggestion
output_dir = fullfile(mrstOutputDirectory, 'PVT', 'PVT_H2STORAGE');
```

Reply:
fixed